src/main/java/teammates/common/util/ActivityLogEntry.java

```diff
@@ -143,6 +155,8 @@ public ActivityLogEntry(String servlet, String act, AccountAttributes acc, Strin
             email = acc.email;
         }
         
+        id = googleId + time;
```

Comment:
Why don't you use `formatCurrentTimeForId() ` here like the line 217 below? Likewise for the line 115.